modules/publicize/publicize-jetpack.php

```diff
@@ -447,7 +447,11 @@ function test_connection( $service_name, $connection ) {
 	 * Save a flag locally to indicate that this post has already been Publicized via the selected
 	 * connections.
 	 */
-	function save_publicized( $post_ID, $post, $update ) {
+	function save_publicized( $post_ID = null, $post = null, $update = null ) {
```

Comment:
@lezama, @ebinnion and @gravityrail Have you tried replicating the issue with the plugin. Should I write up some test to explain what is going on in order to get this merged?